src/services/affinidi/auth/index.ts

```diff
@@ -178,6 +178,6 @@ export class Auth implements AuthSDK {
    * @returns {string} - A value of PrincipalId
    */
   public getPrincipalId(): string {
-    return tokenService.getPrincipalId()
+    return tokenService.getPrincipal()?.id
```

Comment:
The return type of this method is string, but this can return undefined as well

Reply:
I have created a seperate ticket for this : https://lemmatree.atlassian.net/browse/FTL-12384